examples/turbulent_advection-diffusion.jl

```diff
@@ -33,6 +34,7 @@ nothing # hide
       n = 128            # 2D resolution = n²
 stepper = "FilteredRK4"  # timestepper
      dt = 2.5e-3         # timestep
+nothing # hide
 
 # Physical parameters 
```

Comment:
Would this be clearer as a heading `### Physical parameters`?

Reply:
Yeap